<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mostrarResultados</class>
 <widget class="QWidget" name="mostrarResultados">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Distancia_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>123456789</string>
   </property>
  </widget>
  <widget class="QLabel" name="Monedas">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>123456789</string>
   </property>
  </widget>
  <widget class="QLabel" name="Total">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>123456789</string>
   </property>
  </widget>
  <widget class="QLabel" name="Monedas_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>180</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>123456789</string>
   </property>
  </widget>
  <widget class="QLabel" name="Distancia">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>123456789</string>
   </property>
  </widget>
  <widget class="QLabel" name="Total_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>300</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>123456789</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
